### DISCUSSION TITLE
Reset bandwidth to default for each mode

### DISCUSSION THREAD

**Discussion by f4bpp** [AlexandreRouma/SDRPlusPlus#1129] [Ideas]
Hi Alexandre...
Sometime, I accidentally change the bandwidth and I think it would be nice if we could reset the bandwidth to the standard value with a button beside all the modes (USB, LSB, AM, FM...)

Perhaps this is already available but I didn't find where.

Best regards, David.

**Comment by limaunion** [comment]
+1
